src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -5,63 +5,115 @@ import moment from 'moment';
+          <span className="vads-u-visibility--screen-reader">
+            Visit the landing page for Form number
+          </span>
+          {id}{' '}
+        </dfn>
+        {title}
+        <i
+          className="fas fa-angle-right vads-u-margin-left--0p25"
+          style={{ verticalAlign: 'middle' }}
+        />
+      </a>
+    ) : (
+      <>
+        <dfn>
+          <span className="vads-u-visibility--screen-reader">Form number</span>{' '}
+          {id}{' '}
+        </dfn>
+        {title}
+      </>
+    )}
+  </dt>
+);
+
 const SearchResult = ({ form }) => {
   // Escape early if we don't have the necessary form attributes.
   if (!form?.attributes) {
     return null;
   }
 
+  const {
+    attributes: {
+      formToolUrl,
+      formDetailsUrl,
+      lastRevisionOn,
+      relatedForms,
+      title,
+      url,
+    },
+    id,
+  } = form;
+
   // Derive the download link props.
-  const linkProps = deriveLinkProps(form);
+  const linkProps = deriveLinkPropsFromFormURL(url);
 
   // Derive labels.
-  const pdfLabel = form.attributes.url.toLowerCase().includes('.pdf')
-    ? '(PDF)'
-    : '';
-  const lastRevisionOn = form.attributes.lastRevisionOn
-    ? moment(form.attributes.lastRevisionOn).format('MM-DD-YYYY')
+  const pdfLabel = url.toLowerCase().includes('.pdf') ? '(PDF)' : '';
+  const lastRevision = lastRevisionOn
+    ? moment(lastRevisionOn).format('MM-DD-YYYY')
     : 'N/A';
 
   return (
     <>
-      <dt
-        className="vads-u-padding-top--3 vads-u-margin--0 vads-u-border-top--1px vads-u-border-color--gray-lighter vads-u-font-weight--bold"
-        data-e2e-id="result-title"
-      >
-        <dfn>
-          <span className="vads-u-visibility--screen-reader">Form number</span>{' '}
-          {form.id}
-        </dfn>{' '}
-        {form.attributes.title}
-      </dt>
+      {buildFormTitle(id, formDetailsUrl, title)}
 
       <dd className="vads-u-margin-y--1 vads-u-margin-y--1">
         <dfn className="vads-u-font-weight--bold">Form last updated:</dfn>{' '}
-        {lastRevisionOn}
+        {lastRevision}
       </dd>
 
-      <dd className="vads-u-padding-bottom--3">
-        <a href={form.attributes.url} rel="noreferrer noopener" {...linkProps}>
-          Download VA form {form.id} {pdfLabel}
+      {relatedForms.length > 0 ? (
+        <dd className="vads-u-margin-y--1 vads-u-margin-y--1">
+          <dfn className="vads-u-font-weight--bold">Related to:</dfn>{' '}
+          {relatedForms.join(', ')}
+        </dd>
+      ) : null}
+
+      <dd className="vads-u-margin-bottom--1">
+        <a href={url} rel="noreferrer noopener" {...linkProps}>
+          Download VA form {id} {pdfLabel}
         </a>
       </dd>
+
+      {formToolUrl ? (
+        <a
+          className="usa-button usa-button-secondary vads-u-margin-bottom--3"
+          href={formToolUrl}
+        >
+          Go to online tool{' '}
+          <span className="vads-u-visibility--screen-reader">for {title}</span>
```

Comment:
Should the `form.id`  be included in the screen reader text as well?

Reply:
Oh good catch. It should be in there.